src/Sylius/Bundle/AdminBundle/Resources/config/routing/product_variant.yml

```diff
@@ -42,6 +42,29 @@ sylius_admin_product_variant_create:
                     form: "@SyliusAdmin/ProductVariant/_form.html.twig"
                     breadcrumb: "@SyliusAdmin/ProductVariant/Create/_breadcrumb.html.twig"
 
+sylius_admin_product_variant_create_and_add_another:
+    path: /another-new
```

Comment:
I know, it looks like really rotten :potato:, but for now I have no better idea to handle parametrized redirects with parameters :/ Hope somebody can help me with that 😄